src/PanelTraits/AutoSet.php

```diff
@@ -125,6 +178,18 @@ public function getFieldTypeFromDbColumnType($field)
         }
     }
 
+    // Fix for DBAL not supporting enum
+    public function setDoctrineTypesMapping()
+    {
+        $types = ['enum' => 'string'];
+        $platform = \DB::getDoctrineConnection()->getDatabasePlatform();
+        foreach ($types as $type_key => $type_value) {
+            if (!$platform->hasDoctrineTypeMappingFor($type_key)) {
+                $platform->registerDoctrineTypeMapping($type_key, $type_value);
+            }
```

Comment:
I am not the author for this code but it worked witgout any other issues? See Issue #704.